miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -835,6 +844,16 @@ def set_mop_mode(self, mop_mode: MopMode):
         """Set mop mode setting."""
         return self.send("set_mop_mode", [mop_mode.value])[0] == "ok"
 
+    @command()
+    def mop_intensity(self) -> MopIntensity:
+        """Get mop scrub intensity setting."""
+        return MopIntensity(self.send("get_water_box_custom_mode")[0])
```

Comment:
Please add a check that the model (`self.model`) is the expected and raise an exception otherwise.

Reply:
Just added a simple check to make sure it's not anything besides the S7 since that's the only Roborock version with scrub intensity settings.